jobserv/api/project.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (C) 2017 Linaro Limited
 # Author: Andy Doan <andy.doan@linaro.org>
 
+import json
```

Comment:
Typo in commit message: availble -> available.

Reply:
fixed in force push